papis/bibtex.py

```diff
@@ -140,10 +158,10 @@ def bibtex_to_dict(bibtex):
     """
     from bibtexparser.bparser import BibTexParser
 
-    parser = BibTexParser(common_strings=True)
-    parser.ignore_nonstandard_types = False
-    parser.homogenise_fields = False
-    parser.interpolate_strings = True
+    parser = BibTexParser(common_strings=True,
+            ignore_nonstandard_types=False,
+            homogenize_fields=False,
```

Comment:
Unrelated change, but there was a typo: `homogenize` instead of `homogenise` (see [docs](https://bibtexparser.readthedocs.io/en/master/bibtexparser.html#module-bibtexparser.bparser)).